data_visualization_in_open_source_software/data_visualization.md

```diff
@@ -0,0 +1,265 @@
+
+Often, the best way to get the visualization you want is by finding an existing example that's similar to what you want and then copying that code and modifying it as needed.
+
+<div class = "important">
+Don't expect to become fully "fluent" in either ggplot2 or seaborn! They are complex and powerful systems, with lots and lots of detail. Even experienced programmers frequently rely on google to get their plotting code just right.
+</div>
+
+### Selecting the right plot for your data
+
+There are many kinds of plots available, and in most cases more than one kind of plot would be a reasonable choice for your data.
+
+When selecting a plot type, consider
+
+- the type(s) of variables you wish to plot (especially whether they are continuous or categorical)
+- what idea or story you want your plot to communicate
+
+When you're just getting started with data visualization, it's a good idea to spend as much time as you can examining plots you think work well --- What is it that makes them effective and attractive? How do they communicate information? What can you tell about the underlying data from examining the plot?
+
+Google's Material Design provides a [good overview of most kinds of plots and their typical uses](https://material.io/design/communication/data-visualization.html#selecting-charts). Browsing through that collection and others can help give you ideas of plots to try.
+
+### Data visualization workflow
+
+Here is a good workflow for creating a new data visualization using ggplot2 or seaborn:
+
+1. Spend time carefully [thinking through the plot you want to create](#selecting-the-right-plot-for-your-data). For many people, this means drawing a sketch (a great excuse to keep colored pencils at your desk!). Include as many details as you can, like labels, facets, and scales.
+2. Search (online or through the help documentation) for the basic commands to make the kind of plot you're thinking of (scatterplot, line plot, box plot, etc.). If you're an experienced user, this may be a step you can do without google, at least for plots you make frequently.
+3. Using what you found in step 2, write out code to generate a basic version of your plot. Don't worry about appearance at this stage, the goal is just to get the basic structure down. You'll likely try a few different versions, tweaking the code and re-running it, before you get what you want.
+4. Once you have the structure of the plot in place, turn your attention to appearance (labels, colors, themes, axes, etc.). You'll likely need to return to your search engine to get these details right.
+5. Admire your beautiful visualization! And be grateful all the work you just did is saved in your code, so it will be easy for others (including Future You) to recreate your plot and make other similar ones.
+
+### Galleries of example plots
+
+For both step 2 and 4 in the [data visualization workflow](#data-visualization-workflow), browsing galleries of example plots will save you tons of time.
```

Comment:
again, this felt a bit unnatural to me since I had just left that page. 

Since this section of the content is so short, would it be feasible to include it on the data viz workflow page anyways, under a level 4 heading or something like that?